bundles/org.openhab.binding.velbus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -642,6 +880,84 @@
 		<state readOnly="true" pattern="%.1f %unit%">
 		</state>
 	</channel-type>
+	<channel-type id="hour">
+		<item-type>Number</item-type>
+		<label>Hour</label>
+		<state readOnly="false" min="0" max="23">
+		</state>
+	</channel-type>
+	<channel-type id="minute">
+		<item-type>Number</item-type>
+		<label>Minute</label>
+		<state readOnly="false" min="0" max="59">
+		</state>
+	</channel-type>
+	<channel-type id="counter">
+		<item-type>Number</item-type>
+		<label>Counter</label>
+		<state readOnly="false" min="0">
+		</state>
+	</channel-type>
+	<channel-type id="temperatureSetpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature Setpoint</label>
+		<category>Temperature</category>
+		<state readOnly="false" min="-55" max="63.5" step="0.5" pattern="%.1f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
+		<description>Currently measured voltage</description>
+		<category>Voltage</category>
+		<state readOnly="true" pattern="%.1f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Current</label>
+		<description>Currently measured current</description>
+		<category>Current</category>
+		<state readOnly="true" pattern="%.1f %unit%">
+		</state>
+	</channel-type>
+	<channel-type id="percentage">
+		<item-type>Dimmer</item-type>
```

Comment:
Is this a dimmer or a percentage value? If it's the latter it should be a channel `Number:Dimensionless`

Reply:
Modified in commit 5ca761c